src/expand-uri-template-with-parameters.coffee

```diff
@@ -35,7 +35,7 @@ expandUriTemplateWithParameters = (uriTemplate, parameters) ->
       if Object.keys(parameters).indexOf(uriParameter) is -1
         ambiguous = true
         text = "\nAmbiguous URI parameter in template: #{uriTemplate} " + \
-               "\nParameter not defined in blueprint:" + \
+               "\nParameter not defined in API description: " + \
```

Comment:
According to dictionary full name is `API Description Document`. I don't know if you want to use it or not. Just mentioning it.